src/pause/135EE0.c

```diff
@@ -148,7 +148,19 @@ INCLUDE_ASM(s32, "pause/135EE0", pause_handle_input);
 
 INCLUDE_ASM(s32, "pause/135EE0", pause_cleanup);
 
-INCLUDE_ASM(s32, "pause/135EE0", pause_get_total_equipped_bp_cost);
+s32 pause_get_total_equipped_bp_cost(void) {
+    s32 totalCost = 0;
+    s32 i;
+
+    for (i = 0; i < ARRAY_COUNT(gPlayerData.equippedBadges); i++) {
+        ItemID itemID = gPlayerData.equippedBadges[i];
+        if (itemID != ITEM_NONE) {
```

Comment:
Empty line before the if would be good if you don't mind

Reply:
Done